klippy/extras/bme280.py

```diff
@@ -536,11 +528,13 @@ def data_ready(stat, run_gas):
         try:
             # wait until results are ready
             status = self.read_register('EAS_STATUS_0', 1)[0]
-            while not data_ready(status, run_gas):
+            while status & MEASURE_IN_PROGRESS:
```

Comment:
If we are requesting a gas measurement do we need to check the `GAS_IN_PROGRESS` bit as well?